### PR TITLE
Starting integration tests

### PR DESCRIPTION
**Pull request by jperez999**
This is the first phase of integration tests.

### CODE REVIEW

**Comment by jperez999** [comment]
rerun tests

File: tests/int/test_notebooks.py [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+    with open(script_path, "w") as script:
+        script.write("\n".join(lines))
+    subprocess.check_output([sys.executable, script_path])
+
+    # clear out products
+    if clean_up:
+        shutil.rmtree(output_path)
+
+
+def _get_random_criteo_data(rows):
```
Comment (benfred) [inline tests/int/test_notebooks.py:116]: It doesn't look like you're using _get_random_criteo_data / _get_random_rossmann_data in the integration notebooks tests

File: examples/gpu_benchmark.ipynb [hunk 2]
```diff
@@ -33,10 +50,33 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
    "metadata": {},
    "outputs": [],
    "source": [
+=======
+>>>>>>> 7935f7e9c3302cdf89a1c43ff7315c2a43f206cf
```
Comment (benfred) [inline examples/gpu_benchmark.ipynb:79]: Bad merge on the notebook?

File: examples/optimize_criteo.ipynb [hunk 3]
```diff
@@ -198,9 +215,51 @@
...
+      "\u001b[0;31mKeyboardInterrupt\u001b[0m: "
```
Comment (benfred) [inline examples/optimize_criteo.ipynb:259]: Probably shouldn't have the keyboardinterrupt stack trace rendered here

**Pull request merged**